### PR TITLE
chore(CPlusPlus): add binomial coefficient

### PR DESCRIPTION
**Pull request by ipsitagoel**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**
Description: Given two parameters n and r, this is the C++ code to find the value of the Binomial Coefficient i.e. C(n, r).

**Other information:**
Time Complexity: O(r)
Space Complexity: O(1)

### CODE REVIEW

File: algorithms/CPlusPlus/Maths/binomial-coefficient.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	}
+	return result;
+}
+
+int main()
+{
+	int n;
+	cin >> n;
+	int r;
+	cin >> r;
```
Comment (UG-SEP) [inline algorithms/CPlusPlus/Maths/binomial-coefficient.cpp:33]: Just think if the user doesn't know that whats the program then he/she may confuse that what to enter
To tell the user what to enter using `cout`

**Review by UG-SEP — CHANGES_REQUESTED** [review]
@ipsitagoel Please do to changes as given below:
Feel free to ask any queries related to this `PR`😁

File: algorithms/CPlusPlus/Maths/binomial-coefficient.cpp (hunk 1 above)
Comment (ipsitagoel) [reply algorithms/CPlusPlus/Maths/binomial-coefficient.cpp:33]: Okay!

**Comment by ming-tsai** [comment]
Hi @ipsitagoel,
Any updated?

**Comment by ipsitagoel** [comment]
@ming-tsai Kindly review the changes!

File: algorithms/CPlusPlus/Maths/binomial-coefficient.cpp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	int n;
+	cout << "Enter the value of n: ";
+	cin >> n;
+	int r;
+	cout << "Enter the value of r: ";
+	cin >> r;
+	cout << "The Binomial Coefficient is: "; 
+	cout << findBinCoeff(n, r) << endl;
+	return 0;
+}
```
Comment (UG-SEP) [inline algorithms/CPlusPlus/Maths/binomial-coefficient.cpp:39]: Add an input-output example here:

**Review by UG-SEP — CHANGES_REQUESTED** [review]
Please make suggest changes in your code
Else everything is correct😊

File: algorithms/CPlusPlus/Maths/binomial-coefficient.cpp (hunk 2 above)
Comment (ming-tsai) [reply algorithms/CPlusPlus/Maths/binomial-coefficient.cpp:39]: Welcome back @UG-SEP

Comment (ipsitagoel) [reply algorithms/CPlusPlus/Maths/binomial-coefficient.cpp:40]: @UG-SEP Made the requested changes. Kindly review.

Comment (UG-SEP) [reply algorithms/CPlusPlus/Maths/binomial-coefficient.cpp:40]: > Welcome back @UG-SEP

Thanks you

**Review by UG-SEP — CHANGES_REQUESTED** [review]
Great work

**Review by UG-SEP — APPROVED** [review]
Great

**Comment by ipsitagoel** [comment]
Thank you!

**Pull request merged**